src/components/diagrams/singleLineDiagram/single-line-diagram-content.js

```diff
@@ -380,6 +380,7 @@ function SingleLineDiagramContent(props) {
                 EQUIPMENT_TYPES.LINE,
                 EQUIPMENT_TYPES.TWO_WINDINGS_TRANSFORMER,
                 EQUIPMENT_TYPES.THREE_WINDINGS_TRANSFORMER,
+                EQUIPMENT_TYPES.HVDC_LINE,
```

Comment:
displayBranchMenu -> displayWithOperatingStatusMenu
Because 3WT and HVDC lines aren't branch